### ISSUE TITLE
Open to collaborators for women's opportunities abroad!

### ISSUE THREAD

**Issue by shikha-16** [shikha-16/Women-in-Technology#16]
If you are interested to contribute in the form of organizations/opportunities/scholarships or events abroad that could be added as maybe a separate page to this repository, that would be amazing! Let's expand the community!

**Comment by wannabehacker007** [comment]
Yeah, I think I can help you find opportunities curated for women in the US and Europe regions.

**Comment by shikha-16** [comment]
@wannabehacker007 This issue needs the most help right now. I'll assign this to you so you can start working on it immeditaely. Some of the resources mentioned are applicable abroad so do make sure you see those as well. Suggest changes to the repo accordingly!

**Comment by PeppermintNDaisies** [comment]
Hello! I stumbled upon [this event](https://www.womenwhocode.com/boston/events), hope it helps!

**Comment by shikha-16** [comment]
@wannabehacker007 any updates??

**Comment by Victorybrown** [comment]
What kind of resources are we looking for?

**Comment by shikha-16** [comment]
> What kind of resources are we looking for?

Any scholarship/mentorship programs or women in tech centric events abroad!
